src/Worker/RepairDatabase.php

```diff
@@ -65,5 +66,27 @@ public static function execute()
 				'hash' => Strings::getRandomHex(), 'datetime' => $contact['created']]);
 		}
 		DBA::close($pending_contacts);
+
+		// Ensure that there are no "uri-id", "parent-uri-id" or "thr-parent-id" fields that are NULL
+		$items = DBA::select('item', ['id', 'uri', 'guid'], ["`uri-id` IS NULL"]);
+		while ($item = DBA::fetch($items)) {
+			$uriid = ItemURI::insert(['uri' => $item['uri'], 'guid' => $item['guid']]);
```

Comment:
Does this function return the current `uri-id` for this `uri` if it exists?

Reply:
Yes: https://github.com/friendica/friendica/blob/develop/src/Model/ItemURI.php#L56